setup.py

```diff
@@ -34,6 +34,7 @@
         'pytest-xdist',
         # for documentation
         'autodocsumm',
+        's3fs',
```

Comment:
could you pls put this package in the environment.yml as well, so we manage our deps as much from conda-forge as possible? It's there https://anaconda.org/conda-forge/s3fs

Reply:
also, ideally, this is an actual runtime dep, not for documentation only - but I can do the cleanup before merge

Reply:
It is in environment.yml :)